ISISComputingGroup/ibex_gui
# Ticket2049: Remove synoptic selector on components

### Description of work

Synoptic selection has been removed when editing a component as this is nonsensical. 

### To test

See https://github.com/ISISComputingGroup/IBEX/issues/2049

Ensure that default synoptic can still be selected on a configuration but can no longer be selected on a component.

---

#### Code Review


### Functional Tests


### Final Steps
